### PR TITLE
Option for "Return Type" to be JSON in Relationship Extraction

### PR DESCRIPTION
**Pull request by tanmayb123**
Some developers (like me) prefer JSON over XML, and the relationship extraction API does not allow users to choose their preferred return type.

### CODE REVIEW

File: src/main/java/com/ibm/watson/developer_cloud/relationship_extraction/v1/RelationshipExtraction.java [hunk 1]
```diff
@@ -96,5 +99,18 @@ public Dataset getDataset() {
   public void setDataset(final Dataset dataset) {
     this.dataset = dataset;
   }
+  
+  /**
+   * Sets the returnType.
+   * 
+   * @param returnType the new returnType
+   */
+  public void setReturnType(final REReturnType returnType) {
```
Comment (germanattanasio) [inline src/main/java/com/ibm/watson/developer_cloud/relationship_extraction/v1/RelationshipExtraction.java]: Please don't use acronyms. `ReturnType` is fine.

**Comment by germanattanasio** [comment]
This is looking really good!

**Comment by tanmayb123** [comment]
Thanks! And I've removed the RE acronym from REReturnType.

**Comment by germanattanasio** [comment]
Thanks!

**Pull request merged**